test/integration/context_test.rb

```diff
@@ -423,10 +436,16 @@ def test_nested_lambda_is_called_once
   def test_lambda_in_array_is_called_once
     @global = 0
 
-    @context['callcount'] = [1, 2, proc {
-                                     @global += 1
-                                     @global.to_s
-                                   }, 4, 5]
+    @context['callcount'] = [
+      1,
+      2,
+      proc {
+        @global += 1
+        @global.to_s
+      },
+      4,
+      5
+    ]
```

Comment:
Moving the proc to a local variable avoids having to expand out the otherwise concise array

```suggestion
    p = proc {
      @global += 1
      @global.to_s
    }
    @context['callcount'] = [1, 2, p, 4, 5]
```